src/notebook_utils/automation.py

```diff
@@ -31,13 +32,26 @@ def notebook_to_html_cli():
         help="Directory where the html file will be saved. Defaults to the current directory.",
         default=".",
     )
+    parser.add_argument("-v", "--verbose", action="store_true", help="Output logging details.")
     args = parser.parse_args()
 
+    if args.verbose:
+        _logger.setLevel(logging.INFO)
+
     if args.file is None or args.file.suffix != ".ipynb":
-        _logger.warn("You must specify a valid notebook (.ipynb file) to convert.")
+        _logger.error("You must specify a valid notebook (.ipynb file) to convert.")
+        sys.exit(1)
+
+    _logger.info(f"Converting {args.file}.")
```

Comment:
Logger parameters should use this format to take advantage of skipping string formatting if level is disabled:
```python
_logger.info("Converting %s.", args.file)
```

(other places to edit as well)